packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -675,6 +677,19 @@ abstract class WidgetsBinding extends BindingBase with SchedulerBinding, Gesture
   Element get renderViewElement => _renderViewElement;
   Element _renderViewElement;
 
+  /// The currently active set of [AccessibilityFeatures].
+  ///
+  /// This is initialized the first time [runApp] is called and updated whenever
+  /// the a flag is changed.
```

Comment:
the a => a

Reply:
Fixed, but also moved to renderer binding to fix circular dep